### PR TITLE
Revamp README

### LINKED ISSUE #125: Update README
To reflect current features like forcing generation / parameter set / model run / analysis

### PR DESCRIPTION
**Pull request by sverhoeven**
Fixes eWaterCycle/ewatercycle#125 

Incorporates eWaterCycle/ewatercycle#6 

TODO:
- ~Solve eWaterCycle/ewatercycle-pcrglobwb#4~ Using marrmot m14
- ~Pick year that has forcings with same metadata and observations without NaNs~ using 2010

### CODE REVIEW

**Comment by sverhoeven** [comment]
Pcrglob has some problems, switch to using marrmot

**Comment by sverhoeven** [comment]
README example code now works. 

It is a bit to verbose due required conversions, but we can work on that in eWaterCycle/ewatercycle#69 and eWaterCycle/ewatercycle#190.

File: README.rst [hunk 1]
```diff
@@ -20,6 +20,14 @@ A Python package for running hydrological models.
 .. image:: https://img.shields.io/badge/fair--software.eu-%E2%97%8F%20%20%E2%97%8F%20%20%E2%97%8B%20%20%E2%97%8F%20%20%E2%97%8B-orange
     :target: https://fair-software.eu
 
+
+The eWatercycle package makes it easier to use hydrological models without having intimate knowledge about how to install and run it.
```
Comment (SarahAlidoost) [inline README.rst]:
```suggestion
The eWaterCycle package makes it easier to use hydrological models without having intimate knowledge about how to install and run the models.
```

File: README.rst [hunk 2]
```diff
@@ -20,6 +20,14 @@ A Python package for running hydrological models.
...
+
+* Uses container for running model in a isolated and portable way with `grpc4bmi <https://github.com/eWaterCycle/grpc4bmi>`_
+* Rain and sunshine required for input of the model can be generated by the package using `ESMValTool <https://www.esmvaltool.org/>`_
+* Observation data from `GRDC or USGS <https://ewatercycle.readthedocs.io/en/latest/observations.html>`_
+* Exposes `simple interface <https://ewatercycle.readthedocs.io/en/latest/examples/ewatercycle_api_notebook.html>`_ to quickly get up and running
```
Comment (SarahAlidoost) [inline README.rst]:
```suggestion
* Uses container for running models in an isolated and portable way with `grpc4bmi <https://github.com/eWaterCycle/grpc4bmi>`_
* Generates rain and sunshine required for the model using `ESMValTool <https://www.esmvaltool.org/>`_
* Supports observation data from `GRDC or USGS <https://ewatercycle.readthedocs.io/en/latest/observations.html>`_
* Exposes `simple interface <https://ewatercycle.readthedocs.io/en/latest/examples/ewatercycle_api_notebook.html>`_ to quickly get up and running
```

File: docs/index.rst [hunk 3]
```diff
@@ -6,11 +6,15 @@
...
+
 .. toctree::
    :maxdepth: 2
-   :caption: Contents:
 
-.. include:: ../README.rst
+   observations
 
 Content
 =============
```
Comment (SarahAlidoost) [inline docs/index.rst]: This does not seem to be correct. `observations` under header `User's Guide` should be removed, see the [generated documentation of this PR](https://ewatercycle--164.org.readthedocs.build/en/164/). Also, the header `Content` can be replaced by the header `User's Guide`?

**Review by SarahAlidoost — CHANGES_REQUESTED** [review]
@sverhoeven thanks a lot. We have now marrmot14 as an example, cool. :+1:  I left some comments on the readme text. Also, the index page needs some fixes.

File: docs/index.rst (hunk 3 above)
Comment (sverhoeven) [reply docs/index.rst]: Having 2 tocs make things weird. I dropped `User's guide` toc.

**Review by SarahAlidoost — APPROVED** [review]
Looks very good :+1:

**Comment by sverhoeven** [comment]
Thanks for reviewing

**Pull request merged**